modules/markdown.php

```diff
@@ -6,7 +6,7 @@
  * Sort Order: 31
  * First Introduced: 2.8
  * Requires Connection: No
- * Auto Activate: No
+ * Auto Activate: Yes
```

Comment:
Nope, I think you're right.  The other change is so that current sites get it activated when they upgrade since they'll no longer have access to an interface to allow them to upgrade